tests/testutil/service.go

```diff
@@ -220,7 +239,22 @@ func (ts *TestService) GetRunningPods() string {
 	if err != nil {
 		ts.T.Fatalf("Failed to get running pods:  %v", err.Error())
 	}
-	return out
+
+	kubeResult := strings.Split(strings.TrimSpace(out), "\n")
+	runningPods := kubeResult[:0]
+	for _, pod := range kubeResult {
+		if strings.Contains(pod, "Running") {
+			runningPods = append(runningPods, pod)
+		}
+	}
+	return runningPods
+}
+
+// GenerateLoad queries the endpoint multiple times in order to put load on the service
+func (ts *TestService) GenerateLoad() {
+	HeyCmd(GetHostname(ts.GetAppEndpointURL()), "90s", 130)
+	ts.reload()
+	ts.reloadApp()
```

Comment:
Why not ts.GetEndpointUrl ? Why not pass in full url rather than just hostname ? 
Should probably make time and `c` params.

Reply:
`ts.GetEndpointUrl` returns the endpoint with the version included. I just prefer to use the same endpoint that would show up with a `rio ps` command in this function in case we ever use it to generate load on a service that has different versions with weights > 0.

Time and c will be params.

Using hostname because sometimes `hey` has issues with the https endpoint